project/SotaBuild.scala

```diff
@@ -283,15 +283,19 @@ object Dependencies {
 
   lazy val Rest = Akka ++ Slick
 
-  lazy val AmazonS3 =  "com.amazonaws" % "aws-java-sdk-s3" % "1.10.69"
+  lazy val AmazonS3 =  "com.amazonaws" % "aws-java-sdk-s3" % AWSVersion
 
   val JsonWebSecurity = Seq(
-    "com.advancedtelematic" %% "jw-security-core" % Version.JsonWebSecurity,
-    "com.advancedtelematic" %% "jw-security-jca" % Version.JsonWebSecurity,
-    "com.advancedtelematic" %% "jw-security-akka-http" % Version.JsonWebSecurity
+    "com.advancedtelematic" %% "jw-security-core" % JsonWebSecurityVersion,
+    "com.advancedtelematic" %% "jw-security-jca" % JsonWebSecurityVersion,
+    "com.advancedtelematic" %% "jw-security-akka-http" % JsonWebSecurityVersion
   )
 
-  lazy val Kinesis = "com.amazonaws" % "amazon-kinesis-client" % "1.6.3"
+  lazy val Kinesis = Seq(
```

Comment:
In the past, yes. However, I ran into a runtime error which was only resolved once I added these two explicitly. It appears a json library was moved from Kinesis to the aws-java-sdk or something similar.